model/pipeline/src/main/resources/org/apache/beam/model/pipeline/pipeline.json

```diff
@@ -0,0 +1,320 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+
+  "type": "object",
+
+  "$ref": "#/definitions/Pipeline",
+
+  "definitions": {
+    // A Pipeline contains the entire hierarchical graph including
```

Comment:
Are you allowed to even put comments inside?
To my knowledge this isn't valid.

If we can't put comments in, will we have a neighboring file that describes the reasoning behind the schema?